src/packages/sdk/helpers/build.js

```diff
@@ -0,0 +1,36 @@
+const execa = require('execa')
+const esbuild = require('esbuild')
+const glob = require('glob')
+
+const ESBUILD_DEFAULT = {
+  platform: 'node',
+  target: 'node12',
+  sourcemap: 'external',
+  format: 'cjs',
+  outdir: 'dist',
+  tsconfig: 'tsconfig.build.json'
+}
+
+async function build() {
+  await Promise.all([
+    esbuild.build({
+      ...ESBUILD_DEFAULT,
+      entryPoints: glob.sync('./src/**/*.{j,t}s', {
+        ignore: './src/__tests__/**/*.{j,t}s'
```

Comment:
```suggestion
        ignore: './src/__tests__/**/*'
```

Can probably ignore all files in tests

Reply:
If that makes sense I can suggest that change in the other places too